frontend/src/features/public-form/components/FormFields/FormSectionsContext.tsx

```diff
@@ -73,6 +101,8 @@ export const FormSectionsProvider = ({
   return (
     <FormSectionsContext.Provider
       value={{
+        sectionScrollData,
+        updateSectionScrollData: setVisibleFieldIds,
```

Comment:
should we set this name to what it actually does? for clarity instead of having to look at this context.

suggest `setVisibleFieldIdsForScrollData` haha

Reply:
okie!